linux-packaging-appimage/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+chmod -v a+x /scripts/*.sh
```

Comment:
Shouldn't the files just be commited with the X flag set instead? Or even better, embed that command to the Dockerfile script?